.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,15 @@
+{
+	"image": "mcr.microsoft.com/devcontainers/universal:2",
+	"postCreateCommand": "cd /opt && wget 'https://github.com/wpilibsuite/vscode-wpilib/releases/download/v2023.2.1/vscode-wpilib-2023.2.1.vsix'",
```

Comment:
The latest version is v2023.4.3. Any reason to not use it?

Reply:
I didn't realize there was a new version, so I updated and it works